R/mcmc_sample.R

```diff
@@ -298,31 +297,31 @@ setMethod(
 
               keep <- list()
               # Coverage
-              keep$true_coverage <- sim_is_true_effect_covered(
+              keep$true_coverage <- psborrow2:::sim_is_true_effect_covered(
```

Comment:
I guess this is here because of `future`. Maybe we can make a code path that doesn't need future so it cooperates with `devtools`

Reply:
In my understanding `future` cannot access the dev version so is passing whatever was last built. I'm not sure what you mean by make a code path that doesn't need `future`.